templates/mysqlbackup.sh.erb

```diff
@@ -70,18 +70,18 @@ cleanup
 <% if @file_per_database -%>
 mysql --defaults-file=$TMPFILE -s -r -N -e 'SHOW DATABASES' | while read dbname
 do
-  mysqldump --defaults-file=$TMPFILE --opt --flush-logs --single-transaction \
+  mysqldump --defaults-extra-file=$TMPFILE --opt --flush-logs --single-transaction \
```

Comment:
I don't have any issues with this and I'm not too worried about the installations of version < 5.0.

Any objections @cyberious ?